tada_a_bigoh.py

```diff
@@ -35,6 +37,7 @@
     total_loop_list = []
     sum_of_loops = 0
     used_backfill = tada_arguments.backfill
+    steps = 1
```

Comment:
`steps` is initialized in line 33 from `constants.STEP_START`

Reply:
fixed